FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDeviceAgnosticTests.m

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2013, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+
+@interface FBSnapshotTestCaseDeviceAgnosticTests : FBSnapshotTestCase
+
+@end
+
+@implementation FBSnapshotTestCaseDeviceAgnosticTests
+
+- (void)setUp
+{
+    [super setUp];
+    self.deviceAgnostic = YES;
+    self.recordMode = NO;
+}
+
+- (void)testViewSnapshot
+{
+    UIView *redView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 40, 40)];
+    redView.backgroundColor = [UIColor redColor];
+    FBSnapshotVerifyView(redView, nil);
```

Comment:
Because the `keyWindow` is `nil`... :confounded: Even though the app delegate is certainly [making it key and visible](https://github.com/facebook/ios-snapshot-test-case/blob/master/FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo/FBAppDelegate.m#L22). Any ideas?